src/logging.cc

```diff
@@ -37,7 +37,7 @@ Logger::Logger(const char* file, int line, Severity severity, LogStream* stream)
   (*this) << kSeverities_[severity]
           << time_val
           << std::hex << std::this_thread::get_id() << std::dec << " "
```

Comment:
Change this to decimal to make consistent with glog

Reply:
I just meant change std::hex --> std::dec

Reply:
We don't plan to fully reimplement glog or `absl::logging`, but we can't take a dependency on Abseil just yet, so want to be conformant instead.

Reply:
Right, thanks, I didn't notice `std::hex` there.